README.md

```diff
@@ -24,17 +24,38 @@ This package compromises a TypeScript library meant for interacting with Cisco U
 To install type the following:
 ```shell
 npm i @jweb-development/cisco-umbrella
+yarn add @jweb-development/cisco-umbrella
 ```
 
 ## Use Case
-In order to integrate our current API with Cisco Umbrella's API, we had to design a library capable of managing the integration. To achieve this, the package takes data acquired from our api and 
+In order to integrate our current API with Cisco Umbrella's API, we had to design a library capable of managing the integration. To achieve this, the package takes data acquired from our api to be processed by Cisco Umbrella's API. This package currently supports API integration with `Enforcement`, `Destination Lists`, `Destinations`, and `Organizations`.
 
 ## Documentation
-- [API](/docs/api.md)
-- [Examples]((https://github.com/jweb-development/cisco-umbrella/tree/develop/docs/examples.md))
-- [FAQ]((https://github.com/jweb-development/cisco-umbrella/tree/develop/docs/faq.md))
+- ### [API](/docs/api.md)
+<!-- - [Examples](/docs/examples.md) -->
+<!-- - [FAQ](/docs/faq.md) -->
 
-## Contributors
+## Contributing
+
+### Development
+
+To run a development copy of @jweb-development/cisco-umbrella, type the following:
+
+```bash
+# First we clone the repository
+git clone https://github.com/jweb-development/cisco-umbrella.git
```

Comment:
This is applicable only to those in the jweb org; outside collaborators will need to fork the repository